inventory/sample/group_vars/k8s_cluster/k8s-net-calico.yml

```diff
@@ -19,7 +19,7 @@ calico_cni_name: k8s-pod-network
 # calico_pool_name: "default-pool"
 
 # add default ippool blockSize (defaults kube_network_node_prefix)
-# calico_pool_blocksize: 24
+calico_pool_blocksize: 26
```

Comment:
Is it appropriate to modify the existing env directly when the kubespray version is upgraded?
```
kubelet_max_pods: 250
```

Reply:
I think it's appropriate in most scenarios, but there may be some variables that can't be updated, such as `calico_pool_blocksize`